what4/what4.cabal

```diff
@@ -236,6 +236,17 @@ executable quickstart
     parameterized-utils,
     what4
 
+executable abduct
```

Comment:
Is it also possible to add some test cases for abduction as well? The `abduct` executable here won't be run on CI, so if you want to make sure that this functionality doesn't regress, adding test cases would be the safest bet.

Reply:
Alternatively, perhaps we should just convert this into a test suite. It doesn't contain regression tests per se, but it does at least ensure that all of the relevant machinery works without falling over, which is no small feat.

Reply:
Agreed, working on converting it into a series of tests.